### DISCUSSION TITLE
How to create an open pull request?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#18607] [Pull Requests]
I can't understand how to make a successful PR. I can't use Hacktober either.

**Comment by lethetai2605** [comment]
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request

**Comment by MathiasReker** [comment]
This video demonstrate how to create a pull request: https://www.youtube.com/watch?v=OHV64qh-uyY

**Comment by IvailoPetrov06** [comment]
There how to create pull request:
1.Find a project you want to contribute to.
2. Fork it.
3. Clone it to your local system.
4. Make a new branch.
5. Make your changes.
6. Push it back to your repo.
7. Click the Compare & pull request button.
8. Click Create pull request to open a new pull request.
